src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -1059,10 +1057,9 @@ TEST_F(ConnectionHandlerTest, SessionStarted_WithRpc) {
       mock_connection_handler_observer;
   connection_handler_->set_connection_handler_observer(
       &mock_connection_handler_observer);
-  uint32_t session_key =
-      connection_handler_->KeyFromPair(uid_, start_session_id_);
+
   EXPECT_CALL(mock_connection_handler_observer,
-              OnServiceStartedCallback(device_handle_, session_key, kRpc))
+              OnServiceStartedCallback(device_handle_, _, kRpc))
```

Comment:
@anosach-luxoft The field `start_session_id_` was used uninitialized, so became the `session_key` I followed the tested method calls in order to provide a proper initialization, but my attempts failed. This is why I set expectation like this.